csharp/src/Ice/InputStream.cs

```diff
@@ -303,28 +304,6 @@ public void setLogger(Logger logger)
         _logger = logger;
     }
 
-    /// <summary>
```

Comment:
Removed this flexibility that was probably never used (and definitely not tested in C#).